Flow.Launcher.Infrastructure/StringMatcher.cs

```diff
@@ -225,6 +215,52 @@ public MatchResult FuzzyMatch(string query, string stringToCompare, MatchOption
             return new MatchResult(false, UserSettingSearchPrecision);
         }
 
+        private bool IsAcronym(string stringToCompare, int compareStringIndex)
+        {
+            if (IsAcronymChar(stringToCompare, compareStringIndex) || IsAcronymNumber(stringToCompare, compareStringIndex))
+                return true;
+
+            return false;
+        }
+
+        // When counting acronyms, treat a set of numbers as one acronym ie. Visual 2019 as 2 acronyms instead of 5
+        private bool IsAcronymCount(string stringToCompare, int compareStringIndex)
+        {
+            if (IsAcronymChar(stringToCompare, compareStringIndex))
+                return true;
+
+            if (char.IsNumber(stringToCompare[compareStringIndex]) || char.IsDigit(stringToCompare[compareStringIndex]))
+            {
+                return compareStringIndex switch
+                {
+                    int i when i == 0 => true,
+                    int i when char.IsWhiteSpace(stringToCompare[i - 1]) => true,
+                    _ => false,
+                };
+            }
+
+            return false;
+        }
+
+        private bool IsAcronymChar(string stringToCompare, int compareStringIndex)
+        {
+            if (char.IsUpper(stringToCompare[compareStringIndex]) ||
+                    compareStringIndex == 0 || //0 index means char is the start of the compare string, which is an acronym
+                    compareStringIndex != 0 && char.IsWhiteSpace(stringToCompare[compareStringIndex - 1]))
+                return true;
+
+            return false;
+        }
+
+        private bool IsAcronymNumber(string stringToCompare, int compareStringIndex)
+        {
+            if (char.IsNumber(stringToCompare[compareStringIndex]) ||
```

Comment:
This is dumplicate. IsNumber includes IsDigit. I think both of them will perform similar here, because we shouldn't encouter a situation that will determine a unicode number.
I will change this to check >='0' <='9' to avoid potential outside match from unicode.
![image](https://user-images.githubusercontent.com/45326534/106552059-6d4f8a00-64cb-11eb-94ff-468d9aa89957.png)

Reply:
it's actually from your change, was meaning to ask you why IsDigit is used

Reply:
Ah, it maybe possible that I just see there is a method called char.IsNumber that I know it is used to check number without seeing too much detail.
But anyway, it is somehow outside the acronym that we want.